script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -11,7 +11,7 @@ This time I should ask her something more in-depth.
 [sha:e43438af6544d6f66b171946a10f6a58e5b938b5]{
 -- Page 575, Offset 18015.
 -- 「さっきの死者じゃなくて、もっと危険なヤツ……
-"Instead of the Dead from earlier, let's talk about something more dangerous...
+"Other than the Dead from earlier, there were those much more dangerous guys as well...
```

Comment:
Not sure I agree with that Yusuke. Perhaps if this line were in a vacuum I'd get where you were coming from, but the very next line starts with "The Twenty-Seven Ancestors"

Reply:
Yusuke's reasoning was essentially the point.

That said, yeah replacement here isn't great, in hindsight. I think because of the way its put together in Japanese is actually a bit weird. The way this sentence is constructed is implying action from his previous thought, which doesn't work well in English.

"[I don't want to ask about] the Dead from earlier, but about those dangerous guys [you mentioned before]..."

See brackets for implied dialogue. That's a lot though, but perhaps it can be simplified.

Reply:
A bit of a refactor, but how about sth like
> You mentioned some people more dangerous than the Dead from earlier...

Reply:
Perfect